.github/workflows/release-v2.yml

```diff
@@ -112,28 +111,106 @@ jobs:
 
       # uploads the jars by referencing the release-upload-url from create-release job
       - name: Upload jars
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: shogo82148/actions-upload-release-asset@v1
```

Comment:
updating to a supported action. it uses a variable `token` instead of the environment